README.md

```diff
@@ -9,25 +9,47 @@ HAXM can be built as either a kernel-mode driver for Windows or a kernel
 extension for macOS.
 
 ## Building for Windows
-Prerequisites:
-* [Enterprise WDK (EWDK) 10][ewdk10]
+### Prerequisites
 
-   Alternatively, install [all of the following][wdk10] instead of EWDK 10:
-   * Visual Studio 2015
-   * Windows Driver Kit (WDK) for Windows 10
-   * Windows SDK for Windows 10
+**Option A (Visual Studio)**
+* Visual Studio 2017
```

Comment:
Should the download link be added as other bulletins?
* [Visual Studio][vs] 2017
[vs]: https://www.visualstudio.com/downloads/